src/module.js

```diff
@@ -241,7 +241,6 @@ export default class DailyIframe extends EventEmitter {
   //
 
   static supportedBrowser() {
-    methodNotSupportedInReactNative();
```

Comment:
Callout: this change isn't directly related to TS type declaration coverage. I enabled this method in RN.